src/style.css

```diff
@@ -1 +1,48 @@
 /* add styles here */
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 {
+  .stars__star:first-child {
+    background-image: url(./images/star-active.svg);
+  }
+}
+
+.stars--2 {
+  .stars__star:nth-child(-n + 2) {
+    background-image: url(./images/star-active.svg);
+  }
```

Comment:
CSS does not support nested selectors like this. Consider using a CSS preprocessor such as SASS or LESS if you want to use nested syntax. Alternatively, you can flatten the selectors by combining them, e.g., `.stars--2 .stars__star:nth-child(-n + 2)`.